com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 - Added `try`/`catch` around RPC calls, preventing exception from causing further RPC calls to fail (#1329)
 - Fixed an issue where ServerClientId and LocalClientId could have the same value, causing potential confusion, and also fixed an issue with the UNet where the server could be identified with two different values, one of which might be the same as LocalClientId, and the other of which would not.(#1368)
 - IL2CPP would not properly compile (#1359)
+- Fixed NetworkVariable not calling NetworkSerialize on INetworkSerializable types (#1383)
```

Comment:
Can we move this up into the unreleased section as pre.3 is going to go out today and this will likely land in pre.4 which will include this and a UTP update to pre.8 for the UTP Adapter.

Reply:
Done.